README.md

```diff
@@ -1,59 +1,123 @@
-# EventStoreDB
+[![](https://lh3.googleusercontent.com/G6tLxSbJvFodjR_FHrsXs5WOIls0VfuXkWgv60vbRB0WSuJoe-m1cADCsroUHQgJUQMcwp_HNKCLfiTWuCfVwlT607G8niENuGfq5DsnEmWUx_4Szx3GAWI6X1GKRA5iwv_loW0T75cWCAZsRZm3DL4)](https://www.eventstore.com/)
 
-The open-source, functional database with Complex Event Processing in JavaScript.
+EventStoreDB is an operational database built to store events, powered by the state-transition data model. For developers building true data-driven applications for next-generation use cases.
```

Comment:
```suggestion
EventStoreDB is an operational database built to store events, powered by the state-transition data model, for developers building true data-driven applications for next-generation use cases.
```
otherwise the scd sentence is not a sentence